Data_Structures_Answers.md

```diff
@@ -1,18 +1,27 @@
 Add your answers to the questions below.
 
 1. What is the runtime complexity of your ring buffer's `append` method?
```

Comment:
Great job in your answers, for questions 3 and 4, there is also the possibility of doing this in O(1) constant time, but your answer is still right.